src/components/modules/toolbar/blockSettings.ts

```diff
@@ -229,8 +229,10 @@ export default class BlockSettings extends Module<BlockSettingsNodes> {
    * Add Tool's settings
    */
   private addToolSettings(): void {
-    if (_.isFunction(this.Editor.BlockManager.currentBlock.tool.renderSettings)) {
-      $.append(this.nodes.toolSettings, this.Editor.BlockManager.currentBlock.tool.renderSettings());
+    const settingsElement = this.Editor.BlockManager.currentBlock.renderSettings();
```

Comment:
```suggestion
    const settingsElement = this.Editor.BlockManager.currentBlock?.renderSettings();
```

Reply:
`currentBlock` must be defined there. I'd suggest to leave it without `?` because we will know something is wrong

Reply:
but it will throw an exception if currentBlock is null

Reply:
I meant it cannot be null here, but if it is we will got an exception — that means something is not correct in the code